astropy/time/tests/test_basic.py

```diff
@@ -1056,6 +1056,9 @@ def test_to_datetime():
         assert tz.tzname(dt) == tz_dt.tzname()
     assert np.all(time == forced_to_astropy_time)
 
+    with pytest.raises(ValueError):
+        Time('2015-06-30 23:59:60.000').to_datetime()
```

Comment:
Noted. How do i get `.format((iy, im, id, ihr, imin, isec, ifracsec))` part to compare ?